docs/design/features/byreflike-generics.md

```diff
@@ -7,22 +7,25 @@ Using ByRefLike types in Generic parameters is possible by building upon support
 
 ## Runtime impact
 
-Supporting ByRefLike type as Generic parameters will impact the following IL instructions:
+Supporting ByRefLike types as Generic parameters will impact the following IL instructions.
 
-- `box` &ndash; Types with ByRefLike parameters used in fields cannot be boxed.
+Providing a ByRefLike type to the `box` instruction remains invalid and `InvalidProgramException` will be thrown when detected.
+
```

Comment:
```suggestion
```
It would read better if it is next to the TypeLoadException case and follow the same style.